### ISSUE TITLE
Deploy forms-flow-web customizations to OpenShift

### ISSUE THREAD

**Issue by acoard-aot** [bcgov/nr-epd-digital-services#229]
**Describe the task**
The team have created various forms-flow-web customizations, this will allow us to create custom dashboards, components, etc.  This task is about deploying those changes to OpenShift, instead of the default deployment.

This task is blocked on merging in the forms-flow-web customizations into dev.

**Acceptance Criteria**
- [ ] Create BuildConfig to build Dockerfile, and commit to `openshift/templates/` folder, but it is okay if it is manually `oc apply`'d to our tools environment.  Document any steps in a readme.
- [ ] Update the helm yaml to pull the image from the image built in tools environment

**Additional context**
- Sumesh had some useful tips on this one

**Comment by acoard-aot** [comment]
Putting this back on product backlog for now, as there's some more immediate work with project planning and forms etc.

**Comment by acoard-aot** [comment]
Note: Have identified image pull issue with images when doing OCP docker builds.  The issue is intermittent and has to do with Red Hat registry and images. This is definitely a devops task we'll need to fix, but I'll mark it as Tech Debt and can do in a separate ticket.
